common/src/main/java/org/valkyrienskies/mod/mixin/server/MixinMinecraftServer.java

```diff
@@ -229,6 +230,7 @@ private void postTick(final CallbackInfo ci) {
         // Only drag entities after we have updated the ship positions
         for (final ServerLevel level : getAllLevels()) {
             EntityDragger.INSTANCE.dragEntitiesWithShips(level.getAllEntities());
+            Weather2Compat.INSTANCE.tick(level);
```

Comment:
Can you add an if statement here to only run Weather2Compat if the Weather2 mod is loaded?